src/main/java/HttpServer.java

```diff
@@ -7,16 +7,21 @@
 
   public static void main(String[] args) throws IOException {
```

Comment:
Well, I would think if the Port can't be opened you would want to output to the user that the port can not be open. Right now it will output the stack trace with that error, maybe instead the exception could be caught so that I could make the error message look nice instead of a stack trace.